src/components/Layout/KTopbarUI.vue

```diff
@@ -3,7 +3,7 @@
     <q-btn slot="left" flat class="mobile-only" @click="$emit('toggleSidenav')">
       <i class="fa fa-bars"></i>
     </q-btn>
-    <router-link :to="{name: 'group'}" class="logo">
+    <router-link :to="{name: 'group'}" class="logo desktop-only">
```

Comment:
I tend to use `v-if="!$q.platform.is.mobile"` instead because it prevents rendering, while the `desktop-only` class only hides it via css. But as we don't have performance problems yet, it's fine either way.

Reply:
Yeah, makes sense (especially for a picture that doesn't have to be downloaded then...)